modules/practice/practice.route.js

```diff
@@ -0,0 +1,19 @@
+//Package Imports
+const express  =require("express");
+const path  =require("path");
+
+//const imports
+const { printMessage, createHTML, printName } = require("./practice.controller");
+
+//Creating instances
+const router = express.Router();
+
+router.get('/', printMessage)
+
+router.get('/hello', createHTML)
+
+router.post('/api/post', printName)
```

Comment:
Use a separate file for routes as discussed

Reply:
I have created all routes in a separate practice.route.js file